src/app/components/app-images/app-images.component.html

```diff
@@ -74,14 +98,45 @@
           <div class="content-box image-builder-box" *ngIf="!building">
             <div class="columns is-multiline">
               <div class="column is-12">
+                <div class="content-box approve" *ngIf="approve">
+                  <div class="columns is-multiline">
+                    <div class="column is-12">
+                      <h1>Warning:</h1>
+                    </div>
+                    <div class="column is-12">
+                        <div class="column is-12">
+                          Your docker file contains commands that can corrupt Abstruse to work properly ({{dangerousCommands}}).
```

Comment:
Grammar change:
"Your docker file contains commands that can cause Abstruse to not work correctly ({{dangerousCommands}}).

Reply:
tnx, will fix that...